packages/react/src/components/Table/Table.test.jsx

```diff
@@ -142,6 +141,26 @@ describe('Table', () => {
   });
   afterEach(() => {
     console.error.mockClear();
+
+    mockActions.pagination.onChangePage.mockClear();
+    mockActions.toolbar.onApplyFilter.mockClear();
+    mockActions.toolbar.onToggleFilter.mockClear();
+    mockActions.toolbar.onToggleColumnSelection.mockClear();
+    mockActions.toolbar.onClearAllFilters.mockClear();
+    mockActions.toolbar.onCancelBatchAction.mockClear();
+    mockActions.toolbar.onApplyBatchAction.mockClear();
+    mockActions.toolbar.onApplySearch.mockClear();
+    mockActions.toolbar.onDownloadCSV.mockClear();
+    mockActions.toolbar.onShowRowEdit.mockClear();
+    mockActions.table.onRowSelected.mockClear();
+    mockActions.table.onRowClicked.mockClear();
+    mockActions.table.onRowExpanded.mockClear();
+    mockActions.table.onSelectAll.mockClear();
+    mockActions.table.onChangeSort.mockClear();
+    mockActions.table.onApplyRowAction.mockClear();
+    mockActions.table.onEmptyStateAction.mockClear();
+    mockActions.table.onChangeOrdering.mockClear();
+    mockActions.table.onColumnResize.mockClear();
```

Comment:
Why individually instead of `jest.clearAllMocks()`?

Reply:
I get paid per line ;-) No, you are right there is no good reason, will change.

Reply:
Whoa! Does that deal include snapshot changes, too, because I need your help with future negotiations if so?!?!